.vuepress/config.js

```diff
@@ -280,7 +280,7 @@ module.exports = {
             children: [
                 '/extras/kaslr-fix',
                 '/extras/spoof',
-                '/extras/monterey',
+                '/extras/ventura',
```

Comment:
Thoughts on leaving all previous pages in addition to Ventura?

Reply:
Eh - generally the older pages are worked into the guide by the time the new OS comes out, and the pages are still accessible through GitHub anyway, we don't delete them